chatcommands.py

```diff
@@ -540,6 +540,7 @@ def approve(msg, pr_id):
 
     # Forward this, because checks are better placed in gitmanager.py
     try:
+        has_code_permissions = msg.owner.id in [62118, 66258, 73046, 121520, 145827, 156050]
```

Comment:
At a minimum, this needs to check to verify that the command came from chat.SE, as the IDs are not the same on the other chat servers. Without checking which chat server the command came from, it's giving other, different, users authorization to perform this function on the other chat servers.

[Note: I'm going to have more than this single comment, but I'm marking this as requesting changes at this time.]

Reply:
Done